scalableminds/webknossos
# New worker job: Find Largest Segment ID

wk side for https://github.com/scalableminds/webknossos-worker/pull/114

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Edit a dataset. Should look like this (if the largest segment id is empty):
![image](https://user-images.githubusercontent.com/2486553/198021561-7fc541c0-58ad-45e8-979c-f6bcc47e4809.png)
- Click the Detect button
![image](https://user-images.githubusercontent.com/2486553/198021683-35f61248-d8ab-4404-b504-70ce5c6175b8.png)
![image](https://user-images.githubusercontent.com/2486553/198022674-aa17e1d7-51b0-4d8f-8fb0-5a75f5f1bd59.png)

- Should show a toast:
![image](https://user-images.githubusercontent.com/2486553/198021926-835f3503-03b1-4366-b894-c9ff34eef819.png)
- After reloading:
![image](https://user-images.githubusercontent.com/2486553/198021988-55ff78f5-25d6-43cf-bc73-4b40aa31c438.png)
(the message would typically be something like `Updated largest segment id from 1 to 1470623.`)

It would be even better if the largest segment id was automatically updated in the form, but for long running jobs this wouldn't really matter (as one can close the tab) and the interface of the API doesn't really lend itself to that currently.

### Issues:
- follow-up for #6039 

------
(Please delete unneeded items, merge only when none are left open)
